tests/test_create_command_docs.py

```diff
@@ -0,0 +1,76 @@
+import os
+from unittest import TestCase
+
+from click.testing import CliRunner
+
+from tests.conftest import BASE_DIR
+from utils.create_command_docs import docs
+
+
+class TestCreateCommandDocsCli(TestCase):
```

Comment:
I wonder if we can have a test that it generates the documentation correctly? Not sure how practical that is?

Reply:
This would be quite difficult for our current implementation, but also not sure how practical it would be to test this given this is more of a UI test. I could create a small CLI module for the purpose of testing and we could check the documentation with tests, what are your thoughts?

Reply:
That sounds like a reasonable and not too time consuming suggestion.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/83/commits/08b6f086173ae08a0e1f7ec72507266e97b248bc.